ios/FluentUI/List/ListItem.swift

```diff
@@ -0,0 +1,209 @@
+                footer: Footer = String(),
+                @ViewBuilder leadingView: @escaping () -> LeadingView,
+                @ViewBuilder trailingView: @escaping () -> TrailingView) {
+        self.title = title
+        self.subtitle = subtitle
+        self.footer = footer
+        self.leadingView = leadingView()
+        self.trailingView = trailingView()
+    }
+
+    public var body: some View {
+        tokenSet.update(fluentTheme)
+
+        @ViewBuilder
+        var labelStack: some View {
+            let titleView = Text(title)
+                                .foregroundColor(Color(uiColor: tokenSet[.titleColor].uiColor))
+                                .font(Font(tokenSet[.titleFont].uiFont))
+                                .frame(minHeight: ListItemTokenSet.titleHeight)
+                                .lineLimit(state.titleLineLimit)
+                                .truncationMode(state.titleTruncationMode)
+
+            switch layoutType {
+            case .oneLine:
+                titleView
+            case .twoLines, .threeLines:
+                let subtitleView = Text(subtitle)
+                    .foregroundColor(Color(uiColor: tokenSet[.subtitleColor].uiColor))
+                    .lineLimit(state.subtitleLineLimit)
+                    .truncationMode(state.subtitleTruncationMode)
+                VStack(alignment: .leading, spacing: ListItemTokenSet.labelVerticalSpacing) {
+                    titleView
+                    if layoutType == .twoLines {
+                        subtitleView
+                            .font(Font(tokenSet[.subtitleTwoLinesFont].uiFont))
+                            .frame(minHeight: ListItemTokenSet.subtitleTwoLineHeight)
+                    } else if layoutType == .threeLines {
+                        subtitleView
+                            .font(Font(tokenSet[.subtitleThreeLinesFont].uiFont))
+                            .frame(minHeight: ListItemTokenSet.subtitleThreeLineHeight)
+                        Text(footer)
+                            .foregroundColor(Color(uiColor: tokenSet[.footerColor].uiColor))
+                            .font(Font(tokenSet[.footerFont].uiFont))
+                            .frame(minHeight: ListItemTokenSet.footerHeight)
+                            .lineLimit(state.footerLineLimit)
+                            .truncationMode(state.footerTruncationMode)
+                    }
+                }
+            }
+        }
+
+        @ViewBuilder
+        var accessoryView: some View {
+            if let accessoryType = state.accessoryType,
+               let icon = accessoryType.icon,
+               let iconColor = accessoryType.iconColor(tokenSet: tokenSet, fluentTheme: fluentTheme) {
+                let image = Image(uiImage: icon)
+                    .foregroundColor(Color(uiColor: iconColor))
+                if accessoryType == .detailButton {
+                    SwiftUI.Button {
```

Comment:
Why is the SwiftUI prefix needed?

Reply:
Fluent's UIKit button is also just named `Button`.